db/migrate/20240411113530_rename_candidate_to_trainee.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RenameCandidateToTrainee < ActiveRecord::Migration[7.1]
+  def change
+    RolesMetadataDelegateRegions.where(status: 'candidate_delegate').each do |metadata|
+      metadata.update!(status: RolesMetadataDelegateRegions.statuses[:junior_delegate])
+    end
```

Comment:
You can do a bulk update like so: `RolesMetadataDelegateRegions.where(status: 'old').update_all(status: 'new_status')`. This directly translates to a `UPDATE table SET status = 'new' WHERE status = 'old'` query, instead of issung an `UPDATE WHERE id` style query one by one

Reply:
Oh okay, done. Thanks.